src/main/webapp/app/assessment-shared/assessment-header/assessment-header.component.ts

```diff
@@ -1,26 +1,28 @@
-import { Component, Input, Output, EventEmitter } from '@angular/core';
+import { Component, Input, Output, EventEmitter, OnInit } from '@angular/core';
+import { ActivatedRoute } from '@angular/router';
 import { Result } from 'app/entities/result';
+import { AccountService } from 'app/core';
 
 /**
- * The <jhi-assessment-header> component is shared between the modeling and text assessment interfaces.
+ * The <jhi-assessment-header> component is used in the shared assessment layout.
  * It displays a header bar above the assessment editor with information of locking, as well as offering save/submit/etc buttons.
  * This guarantees a unified look and feel for both interfaces.
- * Depending Components need to perform actions based on the save/submit/cancel/resolveConflict/nextSubmission/goBack outputs.
+ * Depending Components need to perform actions based on the save/submit/cancel/resolveConflict/nextSubmission/navigateBack outputs.
  */
 @Component({
     selector: 'jhi-assessment-header',
     templateUrl: './assessment-header.component.html',
     styleUrls: ['./assessment-header.component.scss'],
 })
-export class AssessmentHeaderComponent {
-    @Input() showBackButton: boolean;
-    @Output() goBack = new EventEmitter<void>();
+export class AssessmentHeaderComponent implements OnInit {
+    showBackButton: boolean;
```

Comment:
At the moment the backButton is always shown. Is a scenario possible in which we don't want to show it?
This makes the component's inner workings less obvious as most parameters are passed through `@Input`s, while this one is passed through the URL.

This also implicity creates a bug, as the showBackButton prop is only taken from the URL on `ngOnInit`.

I would personally vote for removing it.

Reply:
We do not show the back button if you open an Assessment from the "Assessment Dashboard"

<details><summary>Screenshot</summary>
<img width="944" alt="Screen Shot 2019-09-10 at 10 42 54" src="https://user-images.githubusercontent.com/6382716/64598383-ec648f80-d3b7-11e9-9194-4fd0b93441c3.png">
</details>

Reply:
Changed. ✅